pkg/manager/member/pvc_resizer.go

```diff
@@ -460,16 +502,16 @@ func (p *pvcResizer) resizeVolumes(ctx *componentVolumeContext) error {
 				return err
 			}
 
-			klog.V(2).Infof("PVC %s/%s storage request is updated from %s to %s",
-				pvc.Namespace, pvc.Name, currentRequest.String(), quantityInSpec.String())
+			klog.V(2).Infof("PVC %q of TC %q storage request is updated from %s to %s", pvcID, tcID, currentRequest.String(), quantityInSpec.String())
+			break // resize pvc one by one
```

Comment:
With multiple PVC case, is it possible that tikv-1 scales pvc0 and tikv-2 scales pvc1?

Reply:
What we are trying to do is to scale all the PVCs for one Pod only for each component.